src/shared/components/inlineLabels/InlineLabels.tsx

```diff
@@ -114,7 +114,11 @@ class InlineLabels extends Component<Props> {
 
 const mstp = (state: AppState, props: OwnProps) => {
   const labels = getAll<Label>(state, ResourceType.Labels)
-  const selectedLabels = getLabels(state, props.selectedLabelIDs)
+
+  let selectedLabels = []
+  if (props.selectedLabelIDs) {
+    selectedLabels = getLabels(state, props.selectedLabelIDs)
+  }
```

Comment:
instead of this, we can also just add null check in the `getLabels` selector. If we go that route, we just have to make sure this component can handle undefined as `selectedLabels` value

Reply:
can also run a `selectedLabels = getLabels(state, props.selectedLabelIDs??[])??[]` so it'll always have something

Reply:
noted!